### ISSUE TITLE
find specimens which use invalid taxonomy

### ISSUE THREAD

**Issue by dustymc** [ArctosDB/arctos#1701]
Blocked by https://github.com/ArctosDB/arctos/issues/1700

Add (specimen search curatorial pane; maybe a report later) the ability to search for specimens which use taxonomy marked as "invalid."

Is this only for accepted IDs?

**Comment by dustymc** [comment]
duplicate of https://github.com/ArctosDB/arctos/issues/1695
